tutorials/cloud-iot-rtdp/streaming/pom.xml

```diff
@@ -0,0 +1,146 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.google.cloud.solutions</groupId>
+	<artifactId>cloud-iot-rtdp</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<properties>
```

Comment:
If you want to use the Google Cloud DevRel configurations, you can include our parent POM.xml settings:

  <!-- Parent POM defines common plugins and properties. -->
      <parent>
              <groupId>com.google.cloud.samples</groupId>
              <artifactId>shared-configuration</artifactId>
              <version>1.0.6</version>
      </parent>

Then you can run `mvn checkstyle:checkstyle` and can lint.